backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -620,6 +623,35 @@ bool OpenGLSdlGraphicsManager::notifyEvent(const Common::Event &event) {
 				return true;
 			}
 
+#ifdef WIN32
+			// Alt-s creates a screenshot
+			if (event.kbd.keycode == Common::KEYCODE_s) {
+				Common::String filename;
+
+				Common::String screenshotsPath = OSystem_Win32::getScreenshotsPath();
+
+				char fullpath[MAXPATHLEN];
+
+				for (int n = 0;; n++) {
+					SDL_RWops *file;
+
+					filename = Common::String::format("scummvm%05d.bmp", n);
+
+					strcpy(fullpath, screenshotsPath.c_str());
+					strcat(fullpath, filename.c_str());
+
+					file = SDL_RWFromFile(fullpath, "r");
+					if (!file)
+						break;
+					SDL_RWclose(file);
+				}
+
+				saveScreenshot(fullpath);
+				debug("Saved screenshot '%s'", filename.c_str());
+
+				return true;
+		}
+#else
```

Comment:
You can factorise a lot of code between the windows case and the else case. Actually only the line where you get the screenshot path needs to be different.